application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCounter.java

```diff
@@ -0,0 +1,39 @@
+package org.togetherjava.tjbot.features.basic;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.features.Routine;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class MemberCounter implements Routine {
+    private final String wordsInCategory;
```

Comment:
`wordsInCategory` doesn't really make sense for what this is. Let's update this to something like `categoryName` since all this class is doing is changing the name of a category.

Reply:
the `wordsInCategory` basically gets the words that should be present in the category we are finding.

Reply:
Okay, the logic might cause problems. 

If we have the categories: "bot info", "server info" etc. All those channels will be renamed. Instead we should use a `.equals` instead of a `.contains` to avoid that scenario.